Soccar [Client]/src/nl/soccar/ui/fx/MapUiFx.java

```diff
@@ -1,26 +1,75 @@
 package nl.soccar.ui.fx;
 
+import java.awt.Rectangle;
 import javafx.scene.canvas.GraphicsContext;
+import javafx.scene.paint.Color;
 import nl.soccar.library.Map;
+import nl.soccar.library.enumeration.MapType;
+import nl.soccar.ui.DisplayConstants;
 
 /**
  *
  * @author PTS34A
  */
 public class MapUiFx extends DrawableFx<Map> {
 
-    public MapUiFx(GameCanvasFx canvas, Map type) {
-        super(canvas, type);
+    public MapUiFx(GameCanvasFx canvas, Map model) {
+        super(canvas, model);
     }
 
     @Override
     public void update() {
-        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
     }
 
     @Override
     public void draw(GraphicsContext context) {
-        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
+        Map map = super.getModel();
+        Rectangle size = map.getSize();
+
+        double x = size.getX();
+        double y = size.getY();
+
+        double centreX = size.getCenterX();
+        double centreY = size.getCenterY();
+
+        double width = size.getWidth();
+        double height = size.getHeight();
+        
+        double boxPositionY = (height / 2) - (DisplayConstants.BOX_HEIGHT / 2);
+        double goalPositionY = (height / 2) - (DisplayConstants.GOAL_HEIGHT / 2);
+
+        if (map.getMapType() == MapType.GRASSLAND) {

```

Comment:
Het tekenen van de lijnen op het speelveld moet altijd gebeuren, ongeacht of het MapType GRASSLAND is.